vsutil/clips.py

```diff
@@ -131,6 +131,13 @@ def plane(clip: vs.VideoNode, planeno: int, /) -> vs.VideoNode:
     """
     if clip.format.num_planes == 1 and planeno == 0:
         return clip
+
+    if hasattr(clip.std, "SplitPlanes"):
+        planes = clip.std.SplitPlanes()
+        if isinstance(planes, vs.VideoNode):
+            planes = [planes]
+        return planes[planeno]
```

Comment:
```suggestion
    if hasattr(clip.std, "SplitPlanes"):
        planes = clip.std.SplitPlanes()
        return planes if isinstance(planes, vs.VideoNode) else planes[planeno]
```
This can be simplified but actually, if I understand what `SplitPlanes` does, we don't even need to catch this here since we already have a no-op on single-plane clips. (L132-133)

@stuxcrystal when would there be a case that `clip.format.num_planes == 1 and planeno == 0` would be false while SplitPlanes only returns a single plane?

If not:

```suggestion
    if hasattr(clip.std, "SplitPlanes"):
        return clip.std.SplitPlanes()[planeno]
```